### PR TITLE
[UI Alignment] Support new Global Nav

### PR DESCRIPTION
**Pull request by vishwam**
1. Add new colors to support the aligned sticky global nav.
2. Deprecate the clickable mixin - the new UI does not require clickable behavior for buttons.

### CODE REVIEW

File: src/_constants.scss [hunk 1]
```diff
@@ -65,9 +65,15 @@ $layout-thumbnail-size: 100px;
...
 
-$nav-expanded-width: 220px;
-$nav-collapsed-width: $layout-nav-item-height;
+// Media query breakpoints:
+
+$screen-xs: 320px;
+$screen-sm: 480px;
+$screen-md: 640px;
+$screen-lg: 1024px;
+$screen-xl: 1366px;
```
Comment (vincenzocaruso) [inline src/_constants.scss]: is missing the 6th breakpoint for 4k screen?

File: src/light/_color.defaults.scss [hunk 2]
```diff
@@ -146,7 +151,7 @@ $theme-light: map-merge((
     // General Borders
     color-border-rest: $color-grey-300,
     color-border-hover: $color-blue-100,
-    color-border-focus: $color-blue-100,
+    color-border-focus: $color-black,
```
Comment (vincenzocaruso) [inline src/light/_color.defaults.scss]: $color-white

File: src/_constants.scss [hunk 3]
```diff
@@ -65,9 +65,15 @@ $layout-thumbnail-size: 100px;
 $layout-centered-text-width: 400px;
 $layout-masthead-height: 48px;
 $layout-nav-collapsed-width: 48px;
+$layout-nav-expanded-width: 220px;
 $layout-chart-y-axis-width: $gutter-bigger;
 $layout-chart-x-axis-height: $gutter-big;
 $layout-chart-axis-tick-size: 6px;
 
-$nav-expanded-width: 220px;
```
Comment (vincenzocaruso) [inline src/_constants.scss]: are these not used anymore?

Comment (vishwam) [reply src/_constants.scss]: no, turns out no one was using it, Navigation.tsx was using its own variable.

File: src/light/_color.defaults.scss (hunk 2 above)
Comment (vincenzocaruso) [reply src/light/_color.defaults.scss]: sync'd offiline

File: src/_constants.scss (hunk 1 above)
Comment (vishwam) [reply src/_constants.scss]: Waiting for Ted to give me the number, add it in later.

**Pull request merged**